libs/fetcher.client.js

```diff
@@ -184,7 +184,9 @@ function Fetcher(options) {
         corsPath: options.corsPath,
         context: options.context || {},
         contextPicker: options.contextPicker || {},
+        retry: options.retry || null,
         statsCollector: options.statsCollector,
+        unsafeAllowRetry: Boolean(options.unsafeAllowRetry),
```

Comment:
Why is called unsafe?

Reply:
the `unsafeAllowRetry` allows POST requests to be retried. In many cases, however, one would not like to have this feature enabled (since it could lead to duplicated transactions in the back-end). For me, a better name would be `retryOnPost` or something like this so it's clear what is happening (or, perhaps, removing this flag completely since, right now, retry is disabled by default).

Regarding the documentation, it's already in the readme: https://github.com/yahoo/fetchr/blob/081b3c7018d5444b6a35b65e6029b260f97075f0/README.md?plain=1#L477